obstacle_stop_planner_refine/include/obstacle_stop_planner/obstacle_point_cloud.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+#define OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+
+#include <memory>
+#include "pcl/point_types.h"
+#include "pcl_conversions/pcl_conversions.h"
+#include "pcl/point_cloud.h"
+#include "pcl/common/transforms.h"
+#include "pcl/filters/voxel_grid.h"
+#include "tf2_ros/transform_listener.h"
+#include "rclcpp/logger.hpp"
+#include "sensor_msgs/msg/point_cloud2.hpp"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "obstacle_stop_planner/param.hpp"
+
+namespace obstacle_stop_planner
+{
+
+class ObstaclePointCloud
```

Comment:
Could you explain the responsibility of this class?

Reply:
I have divided the classes that handle PointCloud into two. One is a class that treats PointCloud as an object, and the other is a class that manipulates points in PointCloud.
`ObstaclePointCloud` class is the former.

Reply:
hmm, I feel normal functions are enough for this use case.

Reply:
@kenji-miyake Yes, I think this class doesn't have enough responsibility. I deleted ObstaclePointCloud class. 94dc95c